sklearn/utils/extmath.py

```diff
@@ -668,6 +668,11 @@ def cartesian(arrays, out=None):
         2-D array of shape (M, len(arrays)) containing cartesian products
```

Comment:
Could you modify the signature of out with:

```
out : ndarray of shape (M, len(arrays))
        Array containing the cartesian products formed of input arrays.
```

Please edit as the `out` parameter of the Parameters section